src/SFA.DAS.AssessorService.Domain/Consts/CertificateActions.cs

```diff
@@ -8,8 +8,8 @@ public static class CertificateActions
         public const string ConfirmAddress = "ConfirmAddress";
         public const string Date = "Date";
         public const string SendTo = "SendTo";
-        public const string FirstName = "FirstName";
-        public const string LastName = "LastName";
+        public const string GivenNames = "GivenNames";
```

Comment:
@ChrisJWoodcock what is the reason for replacing the FirstName, LastName constants with GivenNames, FamilyName?

Reply:
So that the admin service code matches the assessor code, the most correct code in this case was the version from the admin service.